Ensure you are using an up to date, Anaconda-based Python.

Please access a command line (e.g., Anaconda prompt), and install the dev version of MNE:

`pip install git+https://github.com/mne-tools/mne-python.git`

Test MNE is working:

`python -c "import mne; print(mne.__version__)"`

You should be seeing something like:
`0.17.dev0`
